bin/fresh-restart

```diff
@@ -1,6 +1,6 @@
-#/usr/bin/env bash
-nameb=$(basename $(pwd))
+#!/usr/bin/env bash
+
 docker compose kill
 docker compose rm -vf
-docker volume rm ${nameb}_postgres ${nameb}_seed ${nameb}_sphinx_indexes
+docker volume rm hitobito_postgres hitobito_seed hitobito_sphinx_indexes
```

Comment:
that's a bad idea ... most of us have multiple hitobito instances running, one for every wagon.

Reply:
a `docker compose --volumes` removes those volumes, so you don't need to specify any of this